actions/inference-mock/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
+werkzeug
+click
```

Comment:
For completeness, let's add `pyyaml`, since this action depends on it explicitly.